src/popup/components/InviteItem.vue

```diff
@@ -139,17 +140,19 @@ export default defineComponent({
     const inviteLinkBalance = ref(0);
 
     const link = computed(() => {
-      // sg_ prefix was chosen as a dummy to decode from base58Check
-      const secretKey = (encode(Buffer.from(props.secretKey, 'hex'), Encoding.Signature)).slice(3);
+      // nm_ prefix was chosen as a dummy to decode from base58Check
+      const secretKey = (encode(Buffer.from(props.secretKey), Encoding.Name)).slice(3);
       return new URL(
-        router
-          .resolve({ name: ROUTE_INVITE_CLAIM, params: { secretKey } })
-          .href.replace(/^#/, ''),
+        `${router
+          .resolve({ name: ROUTE_INVITE_CLAIM })
+          .href.replace(/^#/, '')}#${secretKey}`,
```

Comment:
`replace(/^#/, '')` was needed because there was the anchor-based router in Cordova
Isn't there still an anchor-based router there? Would two anchors works correctly there?
```
file:///android_asset/www/index.html#/invite#9oyursU4tq3BuW...
```

Reply:
I tested it, it is working correctly.